skimage/measure/_regionprops.py

```diff
@@ -488,8 +488,9 @@ def _props_to_dict(regions, properties=('label', 'bbox'), separator='-'):
     return out
 
 
-def regionprops_table(label_image, intensity_image=None, cache=True,
-                      properties=('label', 'bbox'), separator='-'):
+def regionprops_table(label_image, intensity_image=None, *, cache=True,
```

Comment:
This is an API change isn't it? The star must be before the new parameter to do not change the behavior.

Reply:
@sciunto this function has not yet been released